package.json

```diff
@@ -15,7 +15,7 @@
   ],
   "author": "Krishnan R",
   "maintainer": "itsjafer",
-  "license": "LGPL-2.1",
+  "license": "Apache 2.0",
```

Comment:
I get a warning from npm because this isn't a valid license: I think it should be 'Apache-2.0'.